File: src/main/java/com/xudong/vam/repository/SelectPackageRepository.java
package com.xudong.vam.repository;

import com.xudong.vam.model.SelectPackage;
import org.springframework.data.repository.CrudRepository;
import org.springframework.stereotype.Repository;

@Repository
public interface SelectPackageRepository extends CrudRepository<SelectPackage, Long> {
    SelectPackage findByName(String name);
}

File: src/main/java/com/xudong/vam/core/repository/SelectPackageRepository.kt
package com.xudong.vam.core.repository

import com.xudong.vam.core.model.SelectPackage
import org.springframework.data.repository.CrudRepository
import org.springframework.stereotype.Repository

@Repository
interface SelectPackageRepository : CrudRepository<SelectPackage, Long> {
    fun findByName(name: String): SelectPackage?
}
